backend/Directory.Build.props

```diff
@@ -16,6 +16,24 @@
     <ItemGroup>
         <PackageReference Include="StyleCop.Analyzers" Version="1.1.118" />
     </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="Microsoft.CodeAnalysis" Version="4.1.0" />
+        <PackageReference Include="Microsoft.CodeAnalysis.Compilers" Version="4.1.0" />
+        <PackageReference Include="Microsoft.CodeAnalysis.CSharp.CodeStyle" Version="4.1.0">
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+            <PrivateAssets>all</PrivateAssets>
+        </PackageReference>
+        <PackageReference Include="Microsoft.Net.Compilers" Version="4.1.0">
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+            <PrivateAssets>all</PrivateAssets>
+        </PackageReference>
+    </ItemGroup>
+    <ItemGroup>
+        <PackageReference Include="Microsoft.Net.Compilers.Toolset" Version="4.1.0">
+            <PrivateAssets>all</PrivateAssets>
+            <IncludeAssets>runtime; build; native; contentfiles; analyzers;</IncludeAssets>
+        </PackageReference>
+    </ItemGroup>
```

Comment:
I'm guessing this is to assist with linting / code formatting errors?

Reply:
That's correct, those packages where needed to get the analyzers working propertly